### PR TITLE
Setup logging with Loki and Promtail

### LINKED ISSUE #169: Setup Loki on the ops server

### PR DESCRIPTION
**Pull request by MalinNordqvist**
## What
We needed to set up logging where we chose to use Loki and Promtail because we already have Grafana set up.

## How
Created config files for Promtail and Loki and added them to the docker-compose file for the ops server. On the web server we added a Promtail config file and added it to the docker-compose file and connected it to Loki on the ops server. 

To see the logs you can go in to Grafana and press explore where you then specify Loki. There is two job query's varlog that is for the opt server and webserver that is for the webserver. 

This PR should close #169 and #170

### CODE REVIEW

File: .infrastructure/monitor-server/loki-config.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
+auth_enabled: false
```
Comment (Pinkvinus) [inline .infrastructure/monitor-server/loki-config.yaml:1]: What's the reason that authentication is disabled?

**Review by Pinkvinus — CHANGES_REQUESTED** [review]

**Review by Pinkvinus — APPROVED** [review]
I've approvedbut please react to my comment

File: .infrastructure/monitor-server/loki-config.yaml (hunk 1 above)
Comment (MalinNordqvist) [reply .infrastructure/monitor-server/loki-config.yaml]: We followed a tutorial but has changed it to true now.

Comment (arnenator) [reply .infrastructure/monitor-server/loki-config.yaml]: Okay so now that it is true... have you tested that it works?

**Review by arnenator — APPROVED** [review]

**Pull request merged**